azure-iot-device/tests/iothub/aio/test_async_clients.py

```diff
@@ -1007,6 +1007,36 @@ def connection_string(self, device_connection_string):
     def sas_token_string(self, device_sas_token_string):
         return device_sas_token_string
 
+    @pytest.fixture
+    def fake_assigned_hub(self):
+        return "DumbledoreArmy.azure-devices.net"
+
+    @pytest.fixture
+    def fake_device_id(self):
+        return "elder_wand"
```

Comment:
I don't believe `fake_assigned_hub` or `fake_device_id` are necessary fixtures, as the values are accessible from the `registration_result`  fixture